### DISCUSSION TITLE
开源凭什么必须是数字时代的新型基础设施？ - 开源之道

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#34] [Announcements]
# 开源凭什么必须是数字时代的新型基础设施？ - 开源之道

https://opensourceway.community/posts/open-source-economic/open-source-as-new-infra/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

20世纪最伟大的思想成果, 其实, 不是别的, 而是 进化论,
正是进化论彻底将人类科学思想从传统哲学体系中解放出来, 
毕竟, 上帝是无论如何无法进化出来的;

而当时, 达而文迟迟不愿发表的原因就在: 一直没找到配合自然选择的自然进化动因 ~ 为什么生物会随机变化? 然后由最适应自然环境那那个变化继续带领族群...

好在, 几乎同时, 格雷戈尔·约翰·孟德尔（德语：Gregor Johann Mendel，1822年7月20日－1884年1月6日）在豌豆杂交研究中, 发现了基因, 这才拼上 进化论 最后一块拼图;

其实, 开源并不是 `新型基础设施`, 而一直是人类计算机技术领域永恒的基因变异动因;
因为, 相比闭源软件, 以市场为导向来探索不同;

自由/开源软件项目, 很多都是 Why not 的产物, 
基本上都是现实生活/工作中遇到问题, 又找不到合适工具时, 自行打造的成果;

IT 和其它行业, 最大的不同就在: 其它行业使用的工具, 都是不自己构建的, 而 IT 行业从头到尾从内到外, 所有过程中的涉及的所有工具, 都是 IT 界自行构建的,
这样带来的好处就是可以自行加速演化, 不用等其它人理解了现实问题后, 再构建工具来给我们使用;

所以, 不用参考什么新时代新设施, 要坚定, 要自信, 开源就是软件行业最核心动力, 几乎所有创造/发明, 最早都是开源世界发起的, 然后, 才被商业软件融合过去, 变成产品...

毕竟, 商业软件企业中产品经理没几个会编程的, 而不会编程的产品经理能设计出什么开创性功能软件来?
